app/i18n/cz/conf.php

```diff
@@ -91,6 +91,7 @@
 		'confirm_enabled' => 'Vyžadovat potvrzení pro akci “označit vše jako přečtené”',
 		'display_articles_unfolded' => 'Ve výchozím stavu zobrazovat články otevřené',
 		'display_categories_unfolded' => 'Ve výchozím stavu zobrazovat kategorie zavřené',
+		'full_title' => 'Show the full title of the article when opened <small>(Title bar might look thicker)</small>',
```

Comment:
Could you please add a ` //TODO` at the end of this line and all others that requite a translation?